app/components/Channels/ChannelSummaryList.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Box } from 'rebass'
+import { ChannelSummaryListItem } from 'components/Channels'
+
+const ChannelSummaryList = ({
+  channels,
+  cryptoCurrency,
+  cryptoCurrencies,
+  setCryptoCurrency,
+  showChannelDetail,
+  ...rest
+}) => (
+  <Box as="article" {...rest}>
+    {channels.map((channelObj, index) => {
+      const channel = Object.prototype.hasOwnProperty.call(channelObj, 'channel')
```

Comment:
Is there a need for `hasOwnProperty` vs `in` here ? If we indeed don't want to take into account proto property then maybe it's worth to add a comment why

Reply:
Have simplified this.